nucleus/validate/client.py

```diff
@@ -11,12 +11,17 @@
 from .data_transfer_objects.scenario_test import CreateScenarioTestRequest
 from .errors import CreateScenarioTestError
 from .eval_functions.available_eval_functions import AvailableEvalFunctions
+from .eval_functions.base_eval_function import BaseEvalFunction
 from .scenario_test import ScenarioTest
 
 SUCCESS_KEY = "success"
 EVAL_FUNCTIONS_KEY = "eval_functions"
 
 
+class InvalidEvaluationCriteria(Exception):
+    pass
```

Comment:
nit: I think this should go in the `errors.py` file

Reply:
Good nit 🙂